lib/vmpooler/pool_manager.rb

```diff
@@ -146,7 +146,7 @@ def check_running_vm vm, pool, ttl
             if (
               (host.runtime) and
               (host.runtime.bootTime)
-              ((((Time.now - host.runtime.bootTime)/60).to_s[/^\d+\.\d{1}/].to_f) > ttl)
+              ((((Time.now - host.runtime.bootTime)/60).to_s[/^\d+\.\d{1}/].to_f) >= ttl)
```

Comment:
I understand the diff just fine, but I'm curious why you need to convert to a string and then back to a float.